blobit-cli/pom.xml

```diff
@@ -34,31 +35,6 @@
     </dependencies>
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
if we remove all the dependencies the .jar won't work anymore

how do we make it work ?

Reply:
It won't work as "standalone" jar but you can run it with blobit services libraries for examples.
Shading his dependencies generate many classloading problems when you import the cli artifact into another project: depending from jar loading order you can load different versions of the same class.
Forcing a specific jar ordering in classpath is not a good option

Reply:
I think that you can configure this plug in in order to set a classifier and not to replace the main artifact.
This way you have the base artifact and pom that is what you want but we still have the Uber jar with all the dependencies

Reply:
It could be done. We keep the old artifact name but we add a new classifier "uber" to keep the uber jar with all shaded dependencies and the "plain" clean one to be used in projects